codegen/model/chart.go

```diff
@@ -0,0 +1,53 @@
+package model
+
+import v1 "k8s.io/api/core/v1"
+
+type Chart struct {
+	Operators []Operator
+
+	Values interface{}
+
+	// goes into the chart.yaml
+	Data Data
+}
+
+type Operator struct {
+	Name       string
+	Deployment Deployment
+	Args       []string
+}
+
+// values for Deployment template
+type Deployment struct {
+	Image     Image                    `json:"image,omitempty"`
+	Resources *v1.ResourceRequirements `json:"resources,omitempty"`
+}
+
+type Image struct {
+	Tag        string        `json:"tag,omitempty"  desc:"tag for the container"`
+	Repository string        `json:"repository,omitempty"  desc:"image name (repository) for the container."`
+	Registry   string        `json:"registry,omitempty" desc:"image prefix/registry e.g. (quay.io/solo-io)"`
```

Comment:
Same as below

Reply:
it's a supra-autopilot concern. this is a low-level function for doing an autopilot generate. we can consider something that runs on top and switches between different sets of variables